File: Easy/2025-12-20-944-Delete-Columns-to-Make-Sorted/solution.js
var minDeletionSize = function(strs) {
    let count = 0;
    const numCols = strs[0].length;

    for (let col = 0; col < numCols; col++) {
        for (let row = 1; row < strs.length; row++) {
            if (strs[row][col] < strs[row - 1][col]) {
                count++;
                break;
            }
        }
    }

    return count;
};

File: Easy/2025-12-20-944-Delete-Columns-to-Make-Sorted/solution.py
class Solution:
    def minDeletionSize(self, strs: List[str]) -> int:
        delete_count = 0
        num_columns = len(strs[0])
        
        for col in range(num_columns):
            for row in range(1, len(strs)):
                if strs[row][col] < strs[row - 1][col]:
                    delete_count += 1
                    break
        
        return delete_count